src/templates/data/__mentorsKyiv.html

```diff
@@ -0,0 +1,617 @@
+{% set
+    AllMentors = [
```

Comment:
Why `PascalCase`? 

Also, pls format this part according to `.editorconfig` while you here. Intend should we equal to two spaces.

Reply:
1. It was late night and I was tired( my bad.
2. I will fix today.